### PR TITLE
Fix image builds

### PR DESCRIPTION
**Pull request by MusicDin**

### CODE REVIEW

File: bin/test-image [hunk 1]
```diff
@@ -75,11 +75,6 @@ if [ "${TYPE}" = "vm" ]; then
 
     INSTANCES="${TEST_IMAGE}"
 
-    # Handle distros needing the agent drive.
```
Comment (tomponline) [inline bin/test-image:78]: Please can you update your commit message so that it has a *why* and not just a *what*? Thanks

Comment (MusicDin) [reply bin/test-image:78]: I've added description to the commit:

RHEL falvored ditros do not ship 9p kernel driver, but LXD mounts the agent over virtiofs.

Comment (tomponline) [reply bin/test-image:78]: Thats not strictly true, we export config drive by 9p and virtiofs.

**Review by tomponline — APPROVED** [review]

File: bin/test-image (hunk 1 above)
Comment (MusicDin) [reply bin/test-image:78]: Yes. What I meant is that even though the RHEL flavor distros do not support 9p, LXD is still able to mount the agent over virtiofs.

**Pull request merged**